.github/workflows/tests.yml

```diff
@@ -35,7 +35,13 @@ jobs:
           profile: minimal
           toolchain: nightly
 
-      - uses: Swatinem/rust-cache@v1
+      - name: Cache .cargo
+        id: parsers-cache
```

Comment:
This id needs to be changed... might cause issues because of conflict otherwise. 🤔

Reply:
Yep.
![image](https://user-images.githubusercontent.com/8050659/150591509-f2f5ae0f-4343-4c29-9e50-fe665a70c4f1.png)